ec2utils/ec2deprecateimg/man/man1/ec2deprecateimg.1

```diff
@@ -0,0 +1,198 @@
+.\" Process this file with
+.\" groff -man -Tascii ec2deprecateimg.1
+.\"
+.TH ec2deprecateimg 1
+.SH NAME
+ec2deprecateimg \- Deprecate images in EC2 by adding tags
+.SH SYNOPSIS
+.B ec2deprecateimg [option]
+.SH DESCRIPTION
+.B ec2deprecateimg
+Deprecate images in EC2 by adding tags to an image. The EC2 platform does
+not support a formal deprecation mechanism. The mechanism implemented by this
+tool is a convention. Unfortunately the tags are not sticky, i.e. not visible
+to others if the image is shared.
+
+Images are tagged with:
+- Deprecated on     -> today's date in YYYYMMDD format
+- Removal date      -> today's date plus the deprecation period specified
+- Replacement image -> The AMI ID and name of the replacement image
+
+The image set as the replacement is removed from the list of potential images
+to be deprecated before any matching takes place. Therefore, the deprecation
+search criteria specified with
+.I --image-name-frag
+or
+.I --image-name-match
+cannot match the replacement image.
+.SH OPTIONS
+.IP "-a --account ACCOUNT_NAME"
+Specifies the account to use to connect to EC2. The account is specified
+in the config file
+.I ~/.ec2utils.conf.
+The account is specified as a section with [account-ACCOUNT_NAME] delimiter.
+The options of the sections are
+.I access_key_id
+.I secret_access_key
+.I ssh_key_name
+.I ssh_private_key
+this allows the program to connect to EC2.
+.IP "-d --deprecation-period NUMBER_OF_MONTHS"
+Specifies the number of months this image will be considered deprecated. After
+the deprecation period expires the image may be removed. The deprecation time
+is added to the date on which the program is executed and forms the value for
+the
+.I Removal date
+tag in the format
+.I YYYYMMDD.
+The default deprecation period is 6 months. 
+.IP "n --dry-run"
+The program will not perform any action. It will provide information on
+.I stdout
+about the action sit would perform.
+.IP "-f --file CONFIG_FILE"
+Specifies the configuration file to use. The default is
+.I ~/.ec2utils.conf
+.IP "--image-id AMI_ID"
+Specify the AMI ID of the image to be marked as deprecated. This option is
+mutually exclusive with the
+.I --image-name,
+.I --image-name-frag,
+and
+.I --image-name-match
+.IP "--image-name IMAGE_NAME"
+Specify the name of the image to be deprecated. The program will look for
+an exact match of the name. This option is mutually exclusive with the
+.I --image-id,
+.I --image-name-frag,
+and
+.I --image-name-match
+options.
+.IP "--image-name-frag IMAGE_NAME_FRAGMENT"
+Specify a section of an image name for the image(s) to be deprecated. Every
+image that matches the name fragment will be marked as deprecated. The
+program will look for an exact match of the name. This option is mutually
```

Comment:
"The program will look for an exact match of the name" seems incongruous here. Copy+Paste error or do I need still more coffee?